.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should probably not commit this file.

Reply:
Sorry. I suppose I ought to add that to the .gitignore then?

Reply:
Yes. Honestly, I thought it was included already.

Reply:
That is, add the whole folder `.vscode`.